ckan/lib/plugins.py

```diff
@@ -166,15 +174,17 @@ def register_group_plugins(map):
                 is_organization = group_controller == 'organization'
```

Comment:
This whole check (the setting of the `is_organization` var) should be up one level, outside the `plugin.is_fallback()` if check, as we always need the var.